lib/tasks/clean_agents_memory.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :agents do
+  desc 'set in_process flag to false for some agents'
+  task clean_in_process_memory: :environment do
+    agents = %W[AppfiguresReviewsAgent ChattermillResponseAgent DelightedAgent SurveyMonkeyAgent TypeformAgent TypeformResponseAgent UsabillaAgent]
+
+    agents.each do |a|
+      puts "Setting #{a} agents"
+      klass = "Agents::#{a}".constantize
```

Comment:
Should we maybe just put constants in the array rather than strings? Or is it not possible?